demo/server/src/modules/datasync/Connector.ts

```diff
@@ -37,6 +38,10 @@ export function connect() {
       }
       if (config.sync.customDataHandlers) {
         const handler = new GlobalMongoDataHandler(mongo);
+        const excludeDays = appConfig.getConfig().sync.excludeOldCompleteWorkOrders;
+        if (excludeDays > 0) {
+          handler.addListFilterModifier(excludeCompleteWorkOrders(excludeDays));
```

Comment:
Awesome way of doing that where users can decide if they want this! 
This needs to be documented.